menu/translations/lxqt-settings_eu.directory

```diff
@@ -0,0 +1 @@
+Name[eu]=LXQt-ren konfigurazioa
```

Comment:
Please add the missing newlines (an empty line at the end) to both files. GitHub complains about them.

Reply:
Done, I just added the missing lines.